backend/src/httpserver/httpserver/modules/database/BitIO API.md

```diff
@@ -31,6 +31,7 @@ Stores friend relations between users.
 ```sql
 CREATE TABLE Friends(user1 text NOT NULL,
                      user2 text NOT NULL,
+                     status INTEGER,
```

Comment:
Is status whether or not something has accepted? Like 0 for no and 1 for yes?

Reply:
Clarified in BitIO API Documentation :)